LARS_versions.gradle

```diff
@@ -5,14 +5,14 @@ ext {
 	jackson_version="2.9.+"
 	javax_json_version="1.0"
 	glassfish_json_version="1.0.+"
-	mongodb_java_version="2.13.0"
+	mongodb_java_version="2.14.+"
```

Comment:
Because the mongo jar version has to be specified exactly in the server.xml, and you have updated to 2.14.3, I think this needs to be exactly 2.14.3 as well. Even though that is a sucky thing to do